src/components/myreserve/myreserve.js

```diff
@@ -1,6 +1,99 @@
+import { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getMyReservations, cancelReservation } from '../../redux/reservations/reservetions';
+import avatarBike from '../../icons/bike-icon.png';
+import './myreserve.css';
+
 function ReservePage() {
```

Comment:
- [ ] In `my reservations` section, I couldn't manage to show the my reservations.

It seems that there is a bug in the API call, or in the Rails server:

![image](https://user-images.githubusercontent.com/57421823/193008762-cf91f232-7f20-4541-a45f-d72cf5774751.png)

Kindly squash this bug so your app works as intended.

Reply:
- At this point, I'm not able to see my reversations. Please work on this. 🐱 
![image](https://user-images.githubusercontent.com/20563252/193062939-e96d2b5e-53df-45d1-964e-355358c785f5.png)